src/Button/Button.tsx

```diff
@@ -425,6 +425,8 @@ export const Button: ComponentWithAs<Props, "button"> = ({
                 alignItems: "center",
                 display: "flex",
                 justifyContent: "center",
+                paddingLeft: "12px",
```

Comment:
These changes need to be applied only when buttons are not icon only. Note that the designs don't have the padding applied for icon-only buttons.

Use the [`iconOnly`](https://github.com/apollographql/space-kit/blob/master/src/Button/Button.tsx#L242) variable to conditionally add this padding.

## Design

![image](https://user-images.githubusercontent.com/5224118/63607757-42a49680-c5a1-11e9-9d54-0f3a0a3fed7a.png)


## This PR

![image](https://user-images.githubusercontent.com/5224118/63607797-59e38400-c5a1-11e9-8e05-cdcaf19ceda3.png)